node/src/Commands.ts

```diff
@@ -2010,3 +2010,27 @@ export function createZMPop(
 
     return createCommand(RequestType.ZMPop, args);
 }
+
+/**
+ * @internal
+ */
+export function createBZMPop(
+    keys: string[],
+    modifier: ScoreFilter,
+    timeout: number,
+    count?: number,
+): command_request.Command {
+    const args: string[] = [
+        timeout.toString(),
+        keys.length.toString(),
+        ...keys,
+        modifier,
+    ];
+
+    if (count !== undefined) {
```

Comment:
```suggestion
    if (count) {
```

Reply:
We can do that with non-number types however, we can't do that with number types because it can be 0 which would be treated as false and behaves differently.